### PR TITLE
Pull 58

### PR DESCRIPTION
**Pull request by trontrytel**

### CODE REVIEW

File: EDMF_Environment.pyx [hunk 1]
```diff
@@ -209,7 +209,7 @@ cdef class EnvironmentVariables:
             if self.QL.values[k] > 1e-8 and self.Area.values[k] > 1e-3:
                 self.cloud_base  = fmin(self.cloud_base,  self.Gr.z_half[k])
                 self.cloud_top   = fmax(self.cloud_top,   self.Gr.z_half[k])
-                self.cloud_cover = fmax(self.cloud_cover, self.cloud_fraction.values[k])
+                self.cloud_cover = fmax(self.cloud_cover, self.Area.values[k])
```
Comment (ilopezgp) [inline EDMF_Environment.pyx]: Shouldn't the second term in `fmax()` be `self.Area.values[k]*self.cloud_fraction.values[k]` ?

File: Turbulence_PrognosticTKE.pyx [hunk 2]
```diff
@@ -535,10 +535,13 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
         self.update_GMV_ED(GMV, Case, TS)
         self.compute_covariance(GMV, Case, TS)
 
-        # update grid-mean cloud fraction
+        # update grid-mean cloud fraction and cloud cover
         for k in xrange(self.Gr.nzg):
+            self.EnvVar.Area.values[k] = 1.0 - self.UpdVar.Area.bulkvalues[k]
```
Comment (ilopezgp) [inline Turbulence_PrognosticTKE.pyx:540]: I don't think an environmental variable should be updated here. We can just use it or use (1-...), but not update it.

File: Variables.pyx [hunk 3]
```diff
@@ -295,28 +295,26 @@ cdef class GridMeanVariables:
             Stats.write_profile('HQTcov_mean',self.HQTcov.values[self.Gr.gw:self.Gr.nzg-self.Gr.gw])
 
         Stats.write_profile('cloud_fraction_mean',self.cloud_fraction.values[self.Gr.gw:self.Gr.nzg-self.Gr.gw])
+        Stats.write_ts('cloud_cover_mean', self.cloud_cover)
```
Comment (ilopezgp) [inline Variables.pyx:298]: Just making sure we have this right: The new cloud_cover from Turbulence_prognosticTKE.pyx corresponds to a minimum overlap of clouds. The previous version was maximum overlap assumption.

**Review by ilopezgp — COMMENTED** [review]
I am not sure about some of the changes, and there are others that I would like to understand better.

File: EDMF_Environment.pyx (hunk 1 above)
Comment (trontrytel) [reply EDMF_Environment.pyx]: done

File: Turbulence_PrognosticTKE.pyx (hunk 2 above)
Comment (ilopezgp) [reply Turbulence_PrognosticTKE.pyx:540]: Discussed and agreed that it should be here. Otherwise Soares does not work with the no-clipping version.

**Review by trontrytel — COMMENTED** [review]
done

File: Variables.pyx (hunk 3 above)
Comment (ilopezgp) [reply Variables.pyx:298]: understood.

**Review by ilopezgp — APPROVED** [review]

**Pull request merged**